playbooks/roles/tarsnap/defaults/main.yml

```diff
@@ -47,10 +47,18 @@ TARSNAP_ARCHIVE_NAME: 'backup'
 # Optional snitch to be sent
 TARSNAP_BACKUP_SNITCH: null
 
-# Backups should be daily at most
-TARSNAP_BACKUP_HOUR: '*'
+# Tarsnap Backup settings. This will be translated to
+# a crontab record. Read https://linux.die.net/man/5/crontab
```

Comment:
Also could link to https://docs.ansible.com/ansible/2.5/modules/cron_module.html